PsfRuntime/CreateProcessHook.cpp

```diff
@@ -153,19 +157,68 @@ BOOL WINAPI CreateProcessFixup(
         processInformation = &pi;
     }
 
-    if (!CreateProcessImpl(
-        applicationName,
-        commandLine,
-        processAttributes,
-        threadAttributes,
-        inheritHandles,
-        creationFlags | CREATE_SUSPENDED,
-        environment,
-        currentDirectory,
-        startupInfo,
-        processInformation))
+    auto appConfig = PSFQueryCurrentAppLaunchConfig(true);
+    bool preventBreakAway = false;
+    auto preventBreakAwayPtr = appConfig->try_get("InPackageContext");
+    if (preventBreakAwayPtr)
+    {
+        preventBreakAway = preventBreakAwayPtr->as_boolean().get();
+    }
+
+    if (preventBreakAway)

```

Comment:
I'm a bit wishy-washy on "preventBreakaway" because preventBreakaway isn't the action we're taking.  This is to launch the process in the container.

NIT: rename to createProcessesInContainer.

@rahul89dadhich what do you think?

Reply:
agree, how about "createProcessInAppContext" ?